test/connect.c

```diff
@@ -263,6 +368,12 @@ int main(int argc, char *argv[])
 		return 1;
 	}
 
+	ret = test_connect_timeout(&ring);
+	if (ret == -1) {
```

Comment:
Let's make it `if (ret)`, because almost all tests here return 0 - success, 1 - failure.

Reply:
I am actually being consistent with all the other tests here.

Reply:
Frankly, I don't really care whether -1 or 1, it's just often 1 around tests. If you think `ret == -1` is better, I don't mind, just please make sure  `test_connect_timeout()` doesn't return 1 (as in hunks I suggested).

Reply:
done